pyproject.toml

```
@@ -1,11 +1,159 @@
 [build-system]
-# Minimum requirements for the build system to execute.
-requires = [
-    "setuptools",
-    "wheel",
-]
+requires = ["setuptools>=64", "setuptools_scm[toml]>=8"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "skrub"
+version = "0.2.0.dev0"
+description = "Prepping tables for machine learning"
+authors = [
+    { name="Patricio Cerda", email="patricio.cerda@inria.fr"},
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
```

```suggestion
    "Development Status :: 4 - Beta",
```

while you're at it can you change this so we can close #860